README.md

```diff
@@ -45,19 +46,45 @@ all plugin directories into your `~/.lightning/plugins` directory. The daemon
 will load each executeable it finds in sub-directories as a plugin. In this case
 you don't need to manage all the `--plugin=...` parameters.
 
-### Pylightning
+### `pyln`
 
-All python plugins depend on the `pylightning` library. It can be given in
-several ways:
+To simplify plugin development you can rely on `pyln-client` for the plugin
+implementation, `pyln-proto` if you need to parse or write lightning protocol
+messages, and `pyln-testing` in order to write tests. These libraries can be
+retrieved in a number of different ways:
 
- - Using `pip` tools: `pip3 install pylightning`
+ - Using `pip` tools: `pip3 install pyln-client pyln-testing`
  - Using the `PYTHONPATH` environment variable to include your clightning's
-   shipped `pylightning` library:
+   shipped `pyln-*` libraries:
 
 ```bash
-PYTHONPATH=/path/to/lightnind/contrib/pylightning lightningd
+export PYTHONPATH=/path/to/lightnind/contrib/pyln-client:/path/to/lightnind/contrib/pyln-testing:$PYTHONGPATH
 ```
 
+### Writing tests
+
+The `pyln-testing` library provides a number of helpers and fixtures to write
+tests. While not strictly necessary, writing a test will ensure that your
+plugin is working correctly against a number of configurations (both with and
+without `DEVELOPER`, `COMPAT` and `EXPERIMENTAL_FEATURES`), and more
+importantly that they will continue to work with newly release versions of
+c-lightning.
+
+Writing a test is as simple as this:
```

Comment:
Maybe also point out that the framework will look for files starting with "test_" and functions with "test_" in those files.

```
To write a test, simply add a python file starting with 'test_' in your plugin's directory and add the test code in a function also starting with 'test_' like this:
```

Reply:
Also, an instruction on howto run the tests locally before committing, would be great (if possible). I fiddled around with `pytest` and PYTHONPATH myself, but it does not launch the regtest daemons and thus runs into connection errors.